requirements/pytorch/base.txt

```diff
@@ -3,7 +3,6 @@
 
 numpy>=1.17.2, <1.23.1
 torch>=1.9.*, <1.13.0
-lightning-lite  # actual version comes from src
```

Comment:
let me see how the version alignment is done :)
ok, the issue is documented in #15489